scylla/src/transport/load_balancing/dc_aware_round_robin.rs

```diff
@@ -11,13 +11,15 @@ use tracing::trace;
 pub struct DcAwareRoundRobinPolicy {
     index: AtomicUsize,
     local_dc: String,
+    include_remote_nodes: bool,
 }
 
 impl DcAwareRoundRobinPolicy {
-    pub fn new(local_dc: String) -> Self {
+    pub fn new(local_dc: String, include_remote_nodes: bool) -> Self {
```

Comment:
I think that it would be better to have a setter method for the include_remote_nodes flag. The name of the method (e.g. set_include_remote_nodes) will give more meaning to the boolean flag when looking at the code that initializes the policy. Moreover, this will reduce the amount of code broken by this change.